frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-popovers.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { TopConsumerPopoverProps } from '@console/dynamic-plugin-sdk';
+import {
+  getTop25ConsumerQueries,
+  OverviewQuery,
+} from '@console/shared/src/promql/cluster-dashboard';
+import ConsumerPopover from '@console/shared/src/components/dashboard/utilization-card/TopConsumerPopover';
+import { ProjectModel, PodModel, NodeModel } from '../../../../models';
+import {
+  humanizeCpuCores,
```

Comment:
```suggestion
  humanizeCPUCores,
```

Reply:
this is widely used function across console code, not a new in this PR so i'd rather avoid changing it here.